doc/source/memory-management.rst

```diff
@@ -281,6 +291,20 @@ To enable object spilling to remote storage (any URI supported by `smart_open <h
         },
     )
 
+Remote storage support is still experimental.
+
+Cluster mode
+------------
+To enable object spilling in multi node clusters:
+
+.. code-block:: bash
+  
+  # Note that `object_spilling_config`'s value should be json format.
+  ray start --head --system-config='{"object_spilling_config":"{\"type\":\"filesystem\",\"params\":{\"directory_path\":\"/tmp/spill\"}}"}'
```

Comment:
Ok for now so that this is doc only, but we should really make this either an env var or cli arg

Reply:
This is inevitable unfortunately. (This is the only way to configure object spilling right now. )